recipes/stac-geoparquet/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "stac-geoparquet" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stac_geoparquet-{{ version }}.tar.gz
+  sha256: a615dde87c36353e856e5162f159294386b462d4be47f31ba89191b6c02edc8e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
```

I think this will fix the CI failure.

```suggestion
  number: 0
  entry_points:
    - pc-geoparquet = stac_geoparquet.cli:main
```